settings.gradle

```diff
@@ -1,4 +1,5 @@
 include ':sqlbrite'
+include ':sqlbrite-contentprovider'
```

Comment:
Hmm.. I thought we said same module as `SqlBrite` but different class? Memory is fuzzy on that conversation.